### PR TITLE
Use correct namespaces for File and Folder

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue # .

### CODE REVIEW

File: administrator/components/com_patchtester/src/Model/PullModel.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 use Joomla\Component\Patchtester\Administrator\GitHub\GitHub;
 use Joomla\Component\Patchtester\Administrator\GithubCredentialsTrait;
 use Joomla\Component\Patchtester\Administrator\Helper\Helper;
-use Joomla\Filesystem\File;
+use Joomla\CMS\Filesystem
```
Comment (dgrammatiko) [inline administrator/components/com_patchtester/src/Model/PullModel.php:25]: @alikon could you also patch the next line `use Joomla\CMS\Filesystem\Folder;`

Comment (alikon) [reply administrator/components/com_patchtester/src/Model/PullModel.php:25]: done 
i was unable to replicate on my local

File: administrator/components/com_patchtester/src/Controller/ResetController.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
 use Joomla\Component\Patchtester\Administrator\Model\PullModel;
 use Joomla\Component\Patchtester\Administrator\Model\PullsModel;
 use Joomla\Component\Patchtester\Administrator\Model\TestsModel;
-use Joomla\Filesystem\File;
+use Joomla\CMS\Filesystem
```
Comment (richard67) [inline administrator/components/com_patchtester/src/Controller/ResetController.php:21]:
```suggestion
use Joomla\CMS\Filesystem;
```
Semicolon missing.

File: administrator/components/com_patchtester/src/Controller/ResetController.php [hunk 3]
```diff
@@ -18,7 +18,7 @@
 use Joomla\Component\Patchtester\Administrator\Model\PullModel;
 use Joomla\Component\Patchtester\Administrator\Model\PullsModel;
 use Joomla\Component\Patchtester\Administrator\Model\TestsModel;
-use Joomla\Filesystem\File;
+use Joomla\CMS\Filesystem;
```
Comment (dgrammatiko) [inline administrator/components/com_patchtester/src/Controller/ResetController.php:21]:
```suggestion
use Joomla\CMS\Filesystem\File;
```

File: administrator/components/com_patchtester/src/Model/PullModel.php [hunk 4]
```diff
@@ -22,8 +22,8 @@
 use Joomla\Component\Patchtester\Administrator\GitHub\GitHub;
 use Joomla\Component\Patchtester\Administrator\GithubCredentialsTrait;
 use Joomla\Component\Patchtester\Administrator\Helper\Helper;
-use Joomla\Filesystem\File;
-use Joomla\Filesystem\Folder;
+use Joomla\CMS\Filesystem
```
Comment (dgrammatiko) [inline administrator/components/com_patchtester/src/Model/PullModel.php:25]:
```suggestion
use Joomla\CMS\Filesystem\File;
```

File: administrator/components/com_patchtester/src/Model/PullModel.php [hunk 5]
```diff
@@ -22,8 +22,8 @@
...
+use Joomla\CMS\Filesystem\Folder
```
Comment (dgrammatiko) [inline administrator/components/com_patchtester/src/Model/PullModel.php:26]:
```suggestion
use Joomla\CMS\Filesystem\Folder;
```

**Review by dgrammatiko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**